twine-upload.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+TWINE_USERNAME=$INPUT_USER \
```

Comment:
I'm pretty sure that we have to wrap all values with double quotes so that env vars with spaces wouldn't break the script.
```suggestion
TWINE_USERNAME="$INPUT_USER" \
```